forc/src/utils/defaults.rs

```diff
@@ -1,11 +1,14 @@
 /// We intentionally don't construct this using [serde]'s default deserialization so we get
 /// the chance to insert some helpful comments and nicer formatting.
 pub(crate) fn default_manifest(project_name: &str, entry_type: &str) -> String {
-    let real_name = whoami::realname();
+    let author = match std::env::var("FORC_INIT_MANIFEST_AUTHOR") {
```

Comment:
Abstract this string into a constant since it's used more than once and is part of the public API.

Reply:
Fixed in [afe7cd6](https://github.com/FuelLabs/sway/pull/1487/commits/afe7cd62ff64983632cf2c9bf4b17be7074c47ad)